packages/appstore/src/components/currency-switcher-card/real/real-account-switcher.tsx

```diff
@@ -4,20 +4,23 @@ import { getStatusBadgeConfig } from '@deriv/account';
 import { StatusBadge, Text } from '@deriv/components';
 import CurrencySwitcherContainer from 'Components/containers/currency-switcher-container';
 import CurrencySwitcherLoader from 'Components/pre-loader/currency-switcher-loader';
-import { useStores } from 'Stores/index';
+import { useStore } from '@deriv/stores';
```

Comment:
```suggestion
import { useStore, observer } from '@deriv/stores';
```
@thisyahlen-deriv Please import `observer` from `@deriv/stores` instead of `mobx-react-lite` 🙏🏻